src/config/reddit/config_gen.py

```diff
@@ -13,11 +13,31 @@
 def config_gen():
     if not os.path.isfile(ENV_FILE):
       # ASK FOR CREDENTIALS
-      
-      CLIENT_ID = input('please input your account client id :')
-      CLIENT_SECRET = input('please input your account client secret :')
-      PASSWORD = input('please input your account password :')
-      USERNAME = input('please input your account username :')
+    def CI():
+        CLIENT_ID = input('please input your account client id :')
+        if len(CLIENT_ID) == 14 :
+            pass
+        else:
+            log.info("client id must be 14 characters, try again")
+            CI()
+    CI()
+    def SI():
+        CLIENT_SECRET = input('please input your account client secret :')
+        if len(CLIENT_SECRET) > 27 :
```

Comment:
my client secret is exactly 27  characters. this line is saying that it needs to be more than 27.  so  should it be like >=? else mine won't pass

Reply:
OH ,ma bad .